muikku-core-plugins/src/main/java/fi/otavanopisto/muikku/plugins/notes/NotesRESTService.java

```diff
@@ -74,16 +83,16 @@ public Response createNote(NoteRestModel note) {
     Note newNote = null;
     
     if (EnvironmentRoleArchetype.STUDENT.equals(loggedUserRole)) {
-      newNote = notesController.createNote(note.getTitle(), note.getDescription(), note.getType(), note.getPriority(), note.getPinned(), sessionController.getLoggedUserEntity().getId());
+      newNote = notesController.createNote(note.getTitle(), note.getDescription(), note.getType(), note.getPriority(), note.getPinned(), sessionController.getLoggedUserEntity().getId(), note.getDueDate());
     } else {
-      newNote = notesController.createNote(note.getTitle(), note.getDescription(), note.getType(), note.getPriority(), note.getPinned(), note.getOwner());
+      newNote = notesController.createNote(note.getTitle(), note.getDescription(), note.getType(), note.getPriority(), note.getPinned(), note.getOwner(), note.getDueDate());
     }
     
     return Response.ok(toRestModel(newNote)).build();
   }
   
   //mApi() call (mApi().notes.note.update(noteId, noteRestModel)
-  // Editable fields are title, description, priority and pinned)
+  // Editable fields are title, description, priority, pinned, dueDate & status)
   @PUT
   @Path ("/note/{NOTEID}")
   @RESTPermit(handling = Handling.INLINE)
```

Comment:
Method assumes caller is logged in, so the annotation should be...

`@RESTPermit (handling = Handling.INLINE, requireLoggedIn = true)`

Check other methods of this service as well.